KsApi/models/GraphBackingTests.swift

```diff
@@ -0,0 +1,45 @@
+@testable import KsApi
+import XCTest
+
+final class GraphBackingTests: XCTestCase {
+  func testGraphBackingDecoding() {
+    let jsonString = """
+    {
+      "backings": {
+        "nodes": [
+          {
+            "errorReason": "no_reason",
+            "status": "errored",
+            "project": {
+              "id": "UHJvamVjdC0yMDQ4MTExNDEw",
+              "name": "Cool project",
+              "slug": "/cool-project"
+            }
+          }
+        ]
+      }
+    }
+    """
+    let data = jsonString.data(using: .utf8)
```

Comment:
If we use the non-failable `Data` initializer then we don't need to force-unwrap later on:

```swift
let data = Data(jsonString.utf8)
```